Analytics/analytics.proto

```diff
@@ -16,19 +16,19 @@ message SalesByProductRequest {
 
 message SalesByProductResponse {
   int32 ProductId = 1;
-  double TotalSales = 2;
+  int32 TotalSales = 2;
 }
 
 // GetTop5Customers 
 message Top5CustomersRequest {}
 message Customer {
   int32 CustomerId = 1;
-  double TotalSpent = 2;
+  int32 TotalSpent = 2;
   string CustomerName = 3;
 }
 
 message Top5CustomersResponse {
-  repeated Customer customer = 1;
+  repeated Customer Customer = 1;
```

Comment:
use snake case in field names in proto files

Reply:
done